src/30_Advanced/ComboBox.html

```diff
@@ -0,0 +1,181 @@
+  }--->
+<!--
+  ## Introduction
+
+  This sample demonstrastes a classic dropdown combo, with a text field which filters the dropdown elements 
+  according to the user's input. The options are rendered via amp-list.
+-->
+<!doctype html>
+<html ⚡>
+<head>
+  <title>ComboBox</title>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/..\30_advanced/combobox/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!-- ## Setup -->
+  <!--
+  The amp-selector component is used for the dropdown list options. The amp-bind component is used for the filtering of the dropdown options
+  according to the user's input. The fontawesome css icons are imported for the arrow icon.
+  -->
+  <script async custom-element="amp-selector" src="https://cdn.ampproject.org/v0/amp-selector-0.1.js"></script>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.min.css">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  
+  <style amp-custom>
+        .hidden, .hidden.select-option {
+            display: none;
+        }
+
+        .arrow-up {
+            transform: rotate(180deg);
+        }
+
+        .product-search{
+            width: 100%;
+            max-width: 350px;
+        }
+
+        .search-container {
+            display: flex;
+            flex-flow: row nowrap;
+            position: relative;
+            margin-top: 20px;
+        }
+
+        .search-submit {
+            font-size: 1.25em;
+            padding-right: 13px;
+        }
+
+        .autosuggest-box {
+            box-shadow: 0px 2px 6px rgba(0,0,0,.3);
+            overflow-y: scroll;
+        }
+
+        input.search-box {
+            padding: 5px 8px;
+            font-size: 1em;
+            background-color: #fff;
+            font-family: Arial,"Helvetica Neue",Helvetica,sans-serif;
```

Comment:
Is there a particular reason why you're using a custom font here?

Reply:
Not really, it was just a mistake. I removed this in the latest commit.